great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -2649,6 +2652,7 @@ def get_batch_list(
         path: Optional[str] = None,
         batch_filter_parameters: Optional[dict] = None,
         batch_spec_passthrough: Optional[dict] = None,
+        batch_request_options: Optional[Union[dict, BatchRequestOptions]] = None,
```

Comment:
I added this parameter for consistency, it's only for fluent batch requests. I'd be open to removing it

Reply:
@donaldheppner The existing method -- along with the utility methods it calls -- is horrifically complex, because it was dealing with the following requirements:
1) Already rich `BatchRequest` set of options;
2) Varied-purpose (`path`, `query`) `RuntimeBatchRequest`;
3) Querying/filtering options (we have many -- slicing, indexing, custom `lambda`, and more); 
4) Splitting and sampling directives.
5) Ability to specify either typed objects or their constituent properties as arguments (allegedly for more ease of use).

The introduction of an FDS specific argument adds only marginally to this complexity, so I support it.  We will just need to make sure that it is handled properly in Usage Statistics (once we prioritize that for FDS).  Thanks.